src/main/java/domain/GamePlayer.java

```diff
@@ -0,0 +1,88 @@
+package domain;
+
+import io.Printer;
+import io.Receiver;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class GamePlayer {
+    Printer printer;
+    Receiver receiver;
+    Generator generator;
+
+    public GamePlayer() {
+        this.printer = new Printer();
+        this.receiver = new Receiver();
+        this.generator = new Generator(10);
+    }
+
+    private final int WINNER_CONDITION = 4;
+
+    public void judgeToMove(Car car, int randomNumber) {
+        if (randomNumber >= WINNER_CONDITION)
+            car.moveForward();
+    }
+
+    public Car[] makeArrayAfterGettingName() {
+        printer.printGeneralMessage("INPUT_NAMEOFCAR");
+        String[] listOfName = receiver.receiveName();
+
+        Car[] cars = new Car[listOfName.length];
+        for (int i = 0; i < listOfName.length; i++)
+            cars[i] = new Car(listOfName[i]);
+
+        return cars;
+    }
+
+    public int makeCountAfterGettingNumber() {
+        printer.printGeneralMessage("INPUT_COUNT");
+        return receiver.receiveNumber();
+    }
+
+    public void launchAllRound(Car[] cars, int countRound) {
+        for (int i = 0; i < countRound; i++) {
+            for (Car j : cars) {
```

Comment:
forEach 문에서는 반복자에게 의미있는 이름을 부여하는 것이 나을 것 같습니다.
```suggestion
            for (Car car : cars) {
```